server.js

```diff
@@ -87,6 +90,18 @@ function makeGuess({ guess, id }, resp) {
   })
 }
 
+function clearGame(resp) {
+  game.reset();
+  players = [];
+  resp.status(200).json({
+    numPlayers: players.length,
+    isGenerator: null,
+    guesses: [],
+    isOver: false,
+    display: []
```

Comment:
Now that i'm thinking about it, I wonder if it would clean up tech debt if we just passed one value, say `newGame`. We could reset both ends based upon this one piece. Then again, I dont know if this means our client side would still be considered "up to date" with the server side. Just something to think about.